examples/shopping_ads/add_performance_max_retail_campaign.rb

```diff
@@ -222,7 +219,6 @@ def create_performance_max_campaign_operation(
         # Set the shopping settings.
         c.shopping_setting = client.resource.shopping_setting do |ss|
             ss.merchant_id = merchant_center_account_id
-            ss.sales_country = sales_country
```

Comment:
This is missing the change to add `INSERT_FEED_LABEL_HERE` from http://tg/2050979

Reply:
Fixed!